lib/utils.pm

```diff
@@ -832,16 +832,29 @@ the second run will update the system.
 sub ssh_fully_patch_system {
     my $remote = shift;
     my $cmd_time = time();
+    my $resolver_option = get_var('PUBLIC_CLOUD_GEN_RESOLVER') ? ' --debug-solver ' : '';
```

Comment:
```suggestion
    my $resolver_option = get_var('PUBLIC_CLOUD_GEN_RESOLVER', '--debug-solver');
```
Wdyt ?  Also, spaces should not be added in the paramter.

Reply:
This reverts the logic what Anton wanted to do

Reply:
Ah, I intended that parameter would contain the option string directly, but it will be 1/0, false/true, instead. 
Anyway the before/after `spaces` I still suggest to remove.

Reply:
done